### PR TITLE
[qa] Add QA script and checks

### PR DESCRIPTION
**Pull request by pandafy**
Adds QA script, `moropy-format` to format/lint the code.
Adds GitHub action to check whether the contributor has properly linted the code or not

### CODE REVIEW

**Review by zerefwayne — DISMISSED** [review]

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+exclude = .git
+max-line-length = 110
+# W503: line break before or after operator
+# W504: line break after or after operator
+# W605: invalid escape sequence
+ignore = W605, W503, W504
+
+[isort]
+known_third_party = django
+known_first_party = openwisp_users, openwisp_utils
```
Comment (zerefwayne) [inline setup.cfg]: This has some code related to openwisp, which needs to be changed.

Comment (pandafy) [reply setup.cfg]: Fixed

File: moropy_bot/bot.py [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 import os
+
 import random
 from discord.ext import commands
 
 import discord
+
 from dotenv import load_dotenv
 
 load_dotenv()
```
Comment (rish07) [inline moropy_bot/bot.py:8]: I think we need to comment this out because there is no env to load.

Comment (pandafy) [reply moropy_bot/bot.py:8]: Will it create an error if `.env` is not present? Can you lookup `dotenv` documentation for this?

**Comment by rish07** [comment]
@pandafy what is Isort check?

**Comment by pandafy** [comment]
It sorts your module imports alphabetically and with some other logic like categorising import from thrid party. You can read more about it here: https://github.com/PyCQA/isort

**Comment by rish07** [comment]
Any idea why it's failing @pandafy ?

**Comment by pandafy** [comment]
@rish07 I didn't know why it was failing, but it is fixed now.

**Comment by rish07** [comment]
Lgtm 👍

**Review by rish07 — COMMENTED** [review]
lgtm

**Pull request merged**